tendrl/ceph_integration/sds_sync/__init__.py

```diff
@@ -239,6 +245,230 @@ def _sync_ec_profiles(self):
             crud = Crud()
             crud.create("ec_profile", attrs)
 
+    def _emit_event(self, severity, resource, curr_value, msg):
+        alert = {}
+        alert['source'] = 'tendrl-ceph-integration'
+        alert['pid'] = os.getpid()
+        alert['time_stamp'] = now().isoformat()
+        alert['alert_type'] = 'status'
+        alert['severity'] = SEVERITIES[severity]
+        alert['resource'] = resource
+        alert['current_value'] = curr_value
+        alert['tags'] = dict(
+            message=msg,
+            cluster_id=NS.tendrl_context.cluster_id,
+            cluster_name=NS.tendrl_context.cluster_name,
+            sds_name=NS.tendrl_context.sds_name,
+            fqdn=socket.getfqdn()
+        )
+        alert['node_id'] = NS.node_context.node_id
+        if not NS.node_context.node_id:
+            return
+        Event(
+            Message(
+                "notice",
+                "alerting",
+                {'message': json.dumps(alert)},
+                node_id=NS.node_context.node_id
```

Comment:
node_id here is not required Message class picks it automatically when its not passed...
In case of collectd plugin. it has no NS object and hence we passed it so that the Message class doesn't do NS.node_context.<..> and fail because NS does not exist..